cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,39 @@
+class TimeLoggingPage {
+  visit() {
+    cy.visit('/time-logging');
+  }
+
+  getUrl() {
+    return cy.url();
+  }
+
+  getPageTitle() {
+    return cy.get('.page__title');
+  }
+
+  getCalendar() {
+    return cy.get('.calendar');
+  }
+
+  getTileForm() {
+    return cy.get('.tile.form');
+  }
+
+  getUserInfoTitle() {
```

Comment:
I think something like `getUserName()` or similar would suit better, don't you agree? :) Because even though the name hides under that 'info title' class, it's not exactly clear what 'info title' means and might bring a little confusion when you see `getUserInfoTitle()` somewhere in your code :)

Reply:
Yes, totally agree, thanks :) Already fixed.